README.org

```diff
@@ -117,6 +117,38 @@ In order to create a hydra, bind it to a key using =ryo-modal-key= or =ryo-modal
                ("q" nil "cancel" :color blue)))
 #+END_SRC
 
+** Use-package keyword
+
+Ryo-modal also provides a use-package keyword: =:ryo-modal=, which is similar to =:bind= in that it
+implies =:defer t= and create autoloads for the bound commands. The keyword is followed by one or
+more key-binding commands, using the same syntax as used by =ryo-modal-keys= as is illustrated by
+the following example:
+
+#+begin_src emacs-lisp
+(use-package modal-test
+  :ensure nil
+  :ryo-modal
+  ;; A list of keywords will be applied to all following keybindings up to the next list of keywords.
```

Comment:
Not entirely clear if you have to start with a list of keywords, but my guess is that you don't?

Reply:
It should indeed work without starting with a list of keywords. I agree that this could be clearer in the example though.